### PR TITLE
Support for the RU date format 

### PR DESCRIPTION
**Pull request by rturovtsev**
Add support:

15 января 2024
25 июля
10 февраля 2024 в 9:30
31.12.2023 23:59
31.12.2023

### CODE REVIEW

**Review by olebedev — APPROVED** [review]
LGTM! Thank you 👍

**Review by olebedev — CHANGES_REQUESTED** [review]
Ah, sorry, the tests seems broken. Please fix the errors. 

Happy to have another look, let me know when you're ready.

**Comment by rturovtsev** [comment]
I'm sorry, I corrected the tests

File: rules/zh/casual_date_test.go
Comment (olebedev) [inline rules/zh/casual_date_test.go:14]: I guess this is occasional, right? If so, can you please move it back

Comment (rturovtsev) [reply rules/zh/casual_date_test.go]: I have restored the changes, although with this version, my tests in zh/casual_date_test are currently not

Comment (rturovtsev) [reply rules/zh/casual_date_test.go]: It seems that the reversion may have been a mistake, as my tests are not passing in zh/casual_date_test. Could we consider removing the reversion to ensure the tests function

Comment (olebedev) [reply rules/zh/casual_date_test.go]: @rturovtsev, then ca you please remove this line then so _all_ the test are passing. 

@RexSkz, thanks for clarification, I was not too sure what is going on on that side of the project. I should have noticed this earlier, my bad.

Comment (rturovtsev) [reply rules/zh/casual_date_test.go:14]: Thank you for your feedback! I have removed the lines that were causing the tests to fail. Now all the tests are passing.

**Review by olebedev — APPROVED** [review]

File: rules/zh/casual_date_test.go
Comment (olebedev) [reply rules/zh/casual_date_test.go:14]: Hey @RexSkz, would you mind to give this PR a tick? We can't merge it without your approval.

**Review by RexSkz — APPROVED** [review]

File: rules/zh/casual_date_test.go
Comment (RexSkz) [reply rules/zh/casual_date_test.go:14]: Sure!

Comment (rturovtsev) [reply rules/zh/casual_date_test.go:14]: @olebedev Hi! Can you tell me if there are plans to create a new release?

Comment (olebedev) [reply rules/zh/casual_date_test.go:14]: Hi there, no plans whatsoever, but I will roll this change out as a new tag soon-ish, in a matter of one or two days, bear with me.

Comment (olebedev) [reply rules/zh/casual_date_test.go:14]: @rturovtsev, here you go https://github.com/olebedev/when/releases/tag/v1.1.0

Comment (rturovtsev) [reply rules/zh/casual_date_test.go:14]: Thank you =)

**Pull request merged**
